css/about.css

```diff
@@ -189,7 +189,7 @@ body {
 .my-main-logo {
   border: 1px solid #d3d3d3;
```

Comment:
- [ ] Give to all your article a fixed height to match the origin design

![la1](https://user-images.githubusercontent.com/57562869/152660054-59e66b6d-9b0e-4967-a925-d486b19f9dec.png)

Reply:
Thank you very much for your comments and suggestions... I will work on it for sure!